style.css

```
@@ -0,0 +1,68 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Arvo:wght@700&family=Cutive+Mono&family=Montserrat:ital,wght@0,100;1,900&family=Recursive:wght@300&family=Roboto+Mono:wght@100&family=Splash&display=swap');
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Arvo', serif;
+}
```

```suggestion
*,
*::before,
*::after {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}

input,
button,
textarea,
select {
  font: inherit;
}
```

CSS reset rule for box-sizing and font should be like this, it would be a better option in case you are using more than one (after/before) or (input/button/textarea). And if you reset css like this then you should remove property related to font from textarea and property related to box-sizing from before element below

---

bit confusing...

---

Actually, I'm saying that if you use CSS reset rule as I suggested then it would be a better option. And please take a look at the CSS reset rules